internal/backends/mcu/stm32h735g.rs

```diff
@@ -99,6 +112,82 @@ pub fn init() {
         dp.OCTOSPI2.octospi_hyperbus_unchecked(config, &ccdr.clocks, ccdr.peripheral.OCTOSPI2);
     let hyperram_ptr: *mut u32 = hyperram.init();
 
+    let _ncs = gpiog.pg6.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _clk = gpiof.pf10.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _dqs = gpiob.pb2.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io0 = gpiod.pd11.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io1 = gpiod.pd12.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io2 = gpioe.pe2.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io3 = gpiod.pd13.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io4 = gpiod.pd4.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io5 = gpiod.pd5.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io6 = gpiog.pg9.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io7 = gpiod.pd7.into_alternate::<10>().speed(High).internal_pull_up(true);
+
+    use stm32h7xx_hal::xspi::*;
+    use OctospiWord as XW;
+
+    let mut octospi =
+        dp.OCTOSPI1.octospi_unchecked(12.MHz(), &ccdr.clocks, ccdr.peripheral.OCTOSPI1);
+
+    octospi.configure_mode(OctospiMode::OneBit).unwrap();
+
+    // RDID Read Identification. Abuses address as instruction phase, but that
+    // works in SPI mode.
+    let mut read: [u8; 3] = [0; 3];
+    octospi.read(0x9F, &mut read).unwrap();
```

Comment:
Is running this command necessary? I thought in the example this shows just how to run that command.

Reply:
maybe it's not necessary, no.